app/views/works/show.html.erb

```diff
@@ -110,14 +110,27 @@
       <p>Curator: <%= User.find(@work.curator_user_id).display_name_safe %></p>
     <% end %>
   <% end %>
-
 </div>
 
-<% if @work.pre_curation_uploads.present? %>
-  <%= render(partial: 'works/uploads') %>
-<% end %>
-
-<%= render(partial: 'works/files') %>
+<section class="uploads">
+  <% if @work.approved? %>
+    <h2 class="h2"><%= t('works.uploads.post_curation.heading') %></h2>
+    <% if @work.post_curation_uploads.present? %>
+      <%= render(partial: 'works/uploads/post_curation_attachments') %>
+    <% end %>
+    <% if @post_curation_s3_resources.present? %>
+      <%= render(partial: 'works/uploads/post_curation_s3_resources') %>
+    <% end %>
+  <% else %>
+    <h2 class="h2"><%= t('works.uploads.pre_curation.heading') %></h2>
+    <% if @work.pre_curation_uploads.present? %>
+      <%= render(partial: 'works/uploads/pre_curation_attachments') %>
+    <% end %>
+    <% if @pre_curation_s3_resources.present? %>
```

Comment:
This should be removed for pre-curation resources (as these are persisted as ActiveStorage::Blob objects)